plugins/rbac/src/components/CreateRole/PermissionPoliciesForm.tsx

```diff
@@ -126,6 +126,8 @@ export const PermissionPoliciesForm = ({
 
   const onAddConditions = (index: number, conditions?: ConditionsData) => {
     setFieldValue(`permissionPoliciesRows[${index}].conditions`, conditions);
+    if (!conditions)
+      setFieldValue(`permissionPoliciesRows[${index}].id`, undefined);
```

Comment:
I don't think we should intentionally set value to `undefined`. I know it's been used everywhere, and we don't have a coding standard to follow. Just want to put it out there for discussion.

Reply:
@ciiay thanks for the feedback but why should we not set a value to undefined if its not needed for some case?